recipe_scrapers/potatorolls.py

```diff
@@ -0,0 +1,61 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import get_yields, normalize_string
+
+
+class PotatoRolls(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "potatorolls.com"
+
+    def author(self):
+        return "Martin's Famous Potato Rolls and Bread"
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
```

Comment:
Let's add the `total_time` method too, despite results being absent for some (but not all) recipes?

```suggestion
    def category(self):
        return self.schema.category()

    def total_time(self):
        return self.schema.total_time()
```